alacritty/src/window.rs

```diff
@@ -202,6 +203,14 @@ impl Window {
                 if let Some(parent_window_id) = config.window.embed {
                     x_embed_window(windowed_context.window(), parent_window_id);
                 }
+                // Perform glClear() on-screen to avoid drawing color over the window
+                // at an incorrect size.
+                // We make GL calls here directly, instead of going through QuadRenderer,
+                // to avoid having to initialize SizeInfo.
+                windowed_context.window().set_visible(true);
+                gl_clear(config.colors.primary.background, config.background_opacity());
+                // Swap buffers to commit glClear() color into the screen.
+                windowed_context.swap_buffers().expect("swap buffers");
```

Comment:
You can safely do that later in display, I don't see any point in doing that during window creation?